ios/Framework/Core/Asset/DisplayScreenlet/AssetDisplayScreenlet.swift

```diff
@@ -15,7 +15,8 @@ import UIKit
 
 /// The AssetDisplayScreenletDelegate protocol defines some methods that you use to manage the
 /// AssetDisplayScreenlet events. All of them are optional.
-@objc public protocol AssetDisplayScreenletDelegate: BaseScreenletDelegate {
+@objc(AssetDisplayScreenletDelegate)
```

Comment:
But if the PR doesn't need a rebase/merge and can be integrated directly this is not a problem. I haven't touched this file and develop will have the latest version.